src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+
+    @Override
+    public String goWork() {
+        return "Bulldozer started its work!";
+    }
+
+    @Override
+    public String stopWork() {
+        return "Bulldozer stopped its work!";

```

Comment:
Similarly, the message for stopping work should indicate that it is a Bulldozer that has stopped working.